codespell_lib/data/dictionary.txt

```diff
@@ -9911,6 +9912,7 @@ intot->into
 intpreter->interpreter
 intputs->inputs
 intrduced->introduced
+intreeg->intrigue
```

Comment:
Can you add intrigued and intriguing too.

Reply:
Done

Reply:
Sorry, I mean intreeged->intrigued etc.

Reply:
Done

Reply:
Can we also have intreeged->intrigued and intreeging->intriguing, I suspect dropping the u is a likely equivalent typo based on the genuine one you've found.

Reply:
done